lib/python/rosie/browser/__init__.py

```diff
@@ -160,6 +160,8 @@
     "\"{0}\". This will permanently delete all branches, " +
     "trunk and any local copy." +
     "\n\nAre you sure you wish to proceed?")
+DIALOG_MESSAGE_SUITE_NOT_RUNNING = ("Cannot launch gcontrol: " +
+                                    "suite {0} does not appear to be running.")
```

Comment:
A minor point: the message is basically the same as what is returned by `SuiteNotRunningError.__str__`. I am fine with this change as long as we think this is acceptable.